ACPs/99-validatorsetmanager-contract/README.md

```diff
@@ -120,15 +120,34 @@ For a full implementation, please see the [Reference Implementation](#reference-
  * validator management, as specified in ACP-77.
  */
 abstract contract ACP99Manager {
-    /// @notice Emitted when an initial validator is registered.
-    event RegisteredInitialValidator(bytes32 indexed validationID, bytes20 indexed nodeID, uint64 weight);
-    /// @notice Emitted when a validator registration to the L1 is initiated.
+    /**
+     * @notice Emitted when an initial validator is registered.
+     * @notice The field index is the index of the initial validator in the conversion data.
+     * This is used along with the subnetID as the ACP-118 justification for
+     * signature requests from P-Chain validators over a L1ValidatorRegistrationMessage
```

Comment:
```suggestion
     * This is used along with the subnetID as the ACP-118 justification in
     * signature requests to P-Chain validators over a L1ValidatorRegistrationMessage
```

It took me a few tries to parse this sentence. I think this is slightly more accurate, but feel free to disagree.